deploy/operator.yaml

```diff
@@ -17,8 +17,8 @@ spec:
     spec:
       serviceAccountName: monstorak
       containers:
-      - image: uchapaga/monstorak:39d27f2
-        imagePullPolicy: Always
+      - image: uchapaga/monstorak:2f30eb8
```

Comment:
Please create a org in docker or quay and don't use local repo for any deployment.

Reply:
Working on it.

Reply:
Fixed.